types/src/front/files.ts

```diff
@@ -191,6 +191,20 @@ export function isSupportedDelimitedTextContentType(
   );
 }
 
+export function extensionsForContentType(
+  contentType: SupportedFileContentType
+): string[] {
+  if (isSupportedPlainTextContentType(contentType)) {
+    return [...supportedPlainText[contentType]];
```

Comment:
why not `return supportedPlainText[contentType];` ?

Reply:
they are const